packages/stateless/components/proposal/MultipleChoiceOption.tsx

```diff
@@ -0,0 +1,230 @@
+import CircleIcon from '@mui/icons-material/Circle'
+import cloneDeep from 'lodash.clonedeep'
+import {
+  Control,
+  FieldArrayMethodProps,
+  FieldErrors,
+  FieldValues,
+  Path,
+  UseFormRegister,
+  useFieldArray,
+  useFormContext,
+} from 'react-hook-form'
+import { useTranslation } from 'react-i18next'
+
+import { SuspenseLoader } from '@dao-dao/stateful'
+import { NewProposalForm } from '@dao-dao/stateful/proposal-module-adapter/adapters/CwdProposalMultiple/types'
+import {
+  Action,
+  ActionKey,
+  ActionKeyAndData,
+  UseDefaults,
+  UseTransformToCosmos,
+} from '@dao-dao/types'
+import { validateRequired } from '@dao-dao/utils'
+
+import { ActionCardLoader, ActionSelector } from '../actions'
+import { Button } from '../buttons/Button'
+import { InputErrorMessage } from '../inputs/InputErrorMessage'
+import { TextAreaInput } from '../inputs/TextAreaInput'
+import { TextInput } from '../inputs/TextInput'
+
+export interface MultipleChoiceOptionData {
+  title: string
+  description: string
+  actionData: ActionKeyAndData[]
+}
+
+export interface MultipleChoiceOptionProps<
+  FV extends FieldValues,
+  FieldName extends Path<FV>
+> {
+  errorsOption: FieldErrors
+  title: FieldName
+  description: FieldName
+  registerOption: UseFormRegister<FV>
+  index: number
+  control: Control<FV>
+  actions: Action[]
+  removeOption: () => void
+  addOption: (
+    value:
+      | Partial<MultipleChoiceOptionData>
+      | Partial<MultipleChoiceOptionData>[],
+    options?: FieldArrayMethodProps | undefined
+  ) => void
+  actionsWithData: Partial<
+    Record<
+      ActionKey,
+      {
+        action: Action
+        transform: ReturnType<UseTransformToCosmos>
+        defaults: ReturnType<UseDefaults>
+      }
+    >
+  >
+}
+
+export const MultipleChoiceOption = <
+  FV extends FieldValues,
+  FieldName extends Path<FV>
+>({
+  title,
+  description,
+  registerOption,
+  index,
+  actions,
+  removeOption,
+  addOption,
+  actionsWithData,
+}: MultipleChoiceOptionProps<FV, FieldName>) => {
+  const { t } = useTranslation()
+
+  const {
+    control,
+    watch,
+    formState: { errors },
+  } = useFormContext<NewProposalForm>()
+
+  const optionActionData = watch(`choices.${index}.actionData`)
+  const { append: appendAction, remove: removeAction } = useFieldArray({
+    name: `choices.${index}.actionData`,
+    control,
+    shouldUnregister: true,
+  })
+
+  return (
+    <div className="flex flex-col gap-4 p-6 pt-5">
+      <div className="rounded-lg bg-background-tertiary ">
+        <div className="flex flex-row items-center justify-between gap-6 border-b border-border-secondary py-4 px-6">
+          <div className="flex grow flex-col">
+            <div className="flex flex-row items-center gap-3">
+              <div
+                style={{
+                  paddingTop: '10px',
+                  width: '50px',
+                  height: 'auto',
+                }}
```

Comment:
inlining styles like this is bad practice, and so is using pixel values because they don't vary based on screen font size. can we replace this with `className="pt-2 h-auto w-12"`